packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -141,7 +141,7 @@ export const DevServerMixin = {
 
         const clientCompiler = app.__compiler.compilers.find((compiler) => compiler.name === CLIENT)
         if (clientCompiler) {
-            app.use(webpackHotMiddleware(clientCompiler))
+            app.use('/__mrt/hmr', webpackHotMiddleware(clientCompiler, {path: '/'}))
         }
```

Comment:
I'm wondering if we want to always have this `/__mrt/hmr` endpoint available, even when hot reloading is disabled... like return a 501 if hot reloading not enabled